config/webpack.dev.config.js

```diff
@@ -14,27 +14,28 @@ module.exports = {
       publicPath: '/dist/'
    },
    module: {
-      rules: [{
-         test: /\.js$/,
-         exclude: /node_modules/,
-         use: ['react-hot-loader', 'babel-loader']
-      },
-      {
-         test: /\.less$/,
-         use: [
-            'style-loader',
-            'css-loader?modules&importLoaders=1&localIdentName=[path]___[name]__[local]___[hash:base64:5]',
-            'resolve-url-loader',
-            'less-loader'
-         ]
-      }]
+      rules: [
+         {
+            test: /\.js$/,
+            exclude: /node_modules/,
+            use: ['react-hot-loader', 'babel-loader']
+         },
+         {
+            test: /\.less$/,
+            use: [
+               'style-loader',
+               'css-loader?modules&importLoaders=1&localIdentName=[path]___[name]__[local]___[hash:base64:5]',
+               'resolve-url-loader',
+               'less-loader'
+            ]
+         }
+      ]
    },
    resolve: {
-      extensions: ['', '.js']
+      extensions: ['.js'],
```

Comment:
@jaredreich Just a heads up, in the upgrade from webpack v1 to v2, there are no blank extensions specified here. Instead they are specified using `enforceExtensions` (below). This was causing the dev build to break.